plugins/sinks/columbus/meta.yaml

```diff
@@ -0,0 +1,12 @@
+name: "Columbus HTTP"
+version: "1.0.0"
+sample_config: |-
+  sinks:
+    - name: columbus
```

Comment:
Let's call it title since this is a more human-readable name. Also, does it make sense to move the proper name also to meta.yaml from the register function? Thoughts? any pros and cons we see?

Reply:
just to make sure, changing `info.name` to `info.title`?

Reply:
> Also, does it make sense to move the proper name also to meta.yaml from the register function? Thoughts? any pros and cons we see?

hmm so we fully depends on the `meta.yaml` for the plugin name as well? we could but we would be fully depend on meta.yaml then, I was thinking meta.yaml is more like an addition rather than essential.

pros:
- meta.yaml holds all information

cons:
- right now plugin info is not being evaluated or calculated unless being required, this saves us startup time. Putting proper name to `meta.yaml` will require us to always build and evaluate info for all plugins because we need that information from `meta.yaml`. And we cannot opt out of it because right now we are registering our plugins via `init` which will be called on every startup.

Reply:
Ok. In that case, lets' rename info.name to info.titile to clear out the diffs.